modules/ROOT/pages/digger/ref_terminology.adoc

```diff
@@ -0,0 +1,20 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: ref_terminology_{digger-service}
+
+[id='{context}_ref_terminology']
+
+= Reference material
+
+This section describes terminology that is associated with {digger-service}.
+
+CI/CD:: Continuous Integration / Continuous Development
+
+Jenkins:: A service used to instrument CI/CD solutions
```

Comment:
Jenkins:: A tool that helps to automate  the software development process, with continuous integration and facilitating continuous delivery.

Reply:
I'd agree this should probably be re-worded, @finp suggestions sounds good to me.